bundles/net.tourbook/src/net/tourbook/importdata/RawDataManager.java

```diff
@@ -93,53 +95,55 @@
 // SET_FORMATTING_OFF
 
    private static final String   COLUMN_FACTORY_CATEGORY_MARKER              = net.tourbook.ui.Messages.ColumnFactory_Category_Marker;
+
    private static final String   COLUMN_FACTORY_GEAR_REAR_SHIFT_COUNT_LABEL  = net.tourbook.ui.Messages.ColumnFactory_GearRearShiftCount_Label;
    private static final String   COLUMN_FACTORY_GEAR_FRONT_SHIFT_COUNT_LABEL = net.tourbook.ui.Messages.ColumnFactory_GearFrontShiftCount_Label;
    private static final String   VALUE_UNIT_CADENCE                          = net.tourbook.ui.Messages.Value_Unit_Cadence;
    private static final String   VALUE_UNIT_CADENCE_SPM                      = net.tourbook.ui.Messages.Value_Unit_Cadence_Spm;
    private static final String   VALUE_UNIT_K_CALORIES                       = net.tourbook.ui.Messages.Value_Unit_KCalories;
    private static final String   VALUE_UNIT_PULSE                            = net.tourbook.ui.Messages.Value_Unit_Pulse;
-
    public static final String    LOG_IMPORT_DELETE_TOUR_FILE                 = Messages.Log_Import_DeleteTourFiles;
+
    public static final String    LOG_IMPORT_DELETE_TOUR_FILE_END             = Messages.Log_Import_DeleteTourFiles_End;
```

Comment:
Fields are reformatted, take time to review your own changes before you commit the code, this is what I do to also prevent this formatting issue

![image](https://user-images.githubusercontent.com/1283445/126028191-2592cad6-48d2-4c37-b369-ed5f50b55d30.png)

Reply:
I typically review my changes but like I have mentioned multiple times now, because PRs are not against main but against a branch that is outdated and can have numerous changes, that creates "noise" and sometimes "hides" the actual changes.

The reformatting might be due to some new rules of Eclipse 2021-06. I have reverted manually.